examples/turn_off_notifications.py

```diff
@@ -0,0 +1,102 @@
+import civis
+import os
+import sys
+
+def bulk_fetch(api_method, user_id):
+  page_num = 1
+  jobs = []
+  # LM: Collecting all the user's jobs up front won't be the most memory efficient, not sure if that's a large enough concern to refactor this or not.
+  # Maybe we should check how many jobs some of our power users have? What was the memory usage of this script when you were running it?
```

Comment:
I think this comment here can be removed, but I am still curious about the performance question - can this script handle large numbers of jobs? I think you mentioned testing it on a couple thousand; I'd be curious as to what the script's memory usage was like. Some people might have more than that.

Reply:
updated to not pull in everything at once

Reply:
Awesome, thanks!